go/vt/wrangler/reparent.go

```diff
@@ -934,27 +970,28 @@ func (wr *Wrangler) emergencyReparentShardLocked(ctx context.Context, ev *events
 	if !ok {
 		return fmt.Errorf("couldn't get master elect %v replication position", topoproto.TabletAliasString(masterElectTabletAlias))
 	}
-	masterElectPos, err := mysql.DecodePosition(masterElectStatus.Position)
+	// Use RelayLogPosition to determine most advanced tablet
+	masterElectPos, err := mysql.DecodePosition(masterElectStatus.RelayLogPosition)
```

Comment:
I have the same suggestion here about looking at both positions whenever both are known.

Reply:
Using RelayLogPosition has been deferred to a future PR.